enclave/CMakeLists.txt

```diff
@@ -52,6 +52,14 @@ if (CMAKE_C_COMPILER_ID MATCHES GNU)
   endif ()
 endif ()
 
+if (CODE_COVERAGE)
```

Comment:
Can we make `add_enclave_library` do the check for CODE_COVERAGE instead of each enclave library having to check for CODE_COVERAGE.

Reply:
That would apply the option to every libraries. Currently the scope is only non-3rdpart ones. However, I can add a cmake function to do this logic.

Reply:
Added a wrapper `enclave_enable_code_coverage`